migrations/2025_02_10_101532_telescope_entries_change_content_type.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Support\Str;
+use RonasIT\TelescopeExtension\Traits\TelescopeTrait;
+
+return new class extends Migration
+{
+    use TelescopeTrait;
+
+    public function up()
+    {
+        if ($this->getDatabaseDriver() === 'pgsql') {
```

Comment:
let's implement another helper method in the trait

```suggestion
        if ($this->isPostgreDriver()) {
```